### DISCUSSION TITLE
We cannot get the AR display effect we want

### DISCUSSION THREAD

**Discussion by a295470799** [google/model-viewer#4516] [Q&A]
### Description

The 3D effect displayed in the .usdz file I uploaded is inconsistent with my preview effect. I cannot find the entrance to upload the texture. Can you give me your workflow?

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Version

- model-viewer: v3.1.1

### Browser Affected

- [ ] Safari

### OS

- [ ] iOS

### AR

- [ ] WebXR

**Comment by elalish** [comment]
We can't really help unless you fill out the issue template - What did you expect? What did you get? Do you have images to describe this? A link where we can reproduce it?
